### DISCUSSION TITLE
How to tune motion config params

### DISCUSSION THREAD

**Discussion by jrkoiter** [blakeblackshear/frigate#997] [Ask A Question]
Is there a good description somewhere on how to tune the [motion config parameters](https://blakeblackshear.github.io/frigate/configuration/advanced#motion)? What is the best process to do this?

**Comment by tubalainen** [comment]
I have been running Frigate since version 0.2 and I am happy with the default parameters. Got five cameras in total.

What do you want to achieve with the tweaking/tuning? Less false-positives? Less bounding boxes due to motion?
Please elaborate - im intrigued!

**Reply by jrkoiter** [reply]
Just trying to avoid starting unnecessary object detection due to weather like wind (moving plants/trees) or rain. Or sudden big changes eg. switching from bright sunlight to shade. I'm currently using [Motion](https://motion-project.github.io/motion_config.html) for this purpose, and it offers a separate view of the video feed that shows where it is detecting motion to assist in fine tuning the parameters to avoid false positives. Wondering if there is something similar in Frigate to help tune the parameters for a specific situation.
